applications/directory/src/components/ProfileReview.js

```diff
@@ -193,7 +238,7 @@ const ProfileReview = ({ onSubmit }) => {
           >
             <Button
               type="submit"
-              disabled={!isValid || !dirty || isSubmitting}
+              disabled={!isValid || !dirty || isSubmitting} // TODO - delete !isDirty because it disables the button for approve cases
```

Comment:
True. Maybe `!dirty` does not belong here. Good catch.

Reply:
will be done in oksana branch